manifests/1.1.0/opensearch-dashboards-1.1.0.yml

```diff
@@ -16,9 +16,10 @@ components:
     working_directory: workbench
     ref: "1.1"
   # hybrid repo (dashboards-notifications)
-  # - name: notificationsDashboards
-  #   repository: https://github.com/opensearch-project/notifications.git
-  #   ref: main
+  - name: notificationsDashboards
+    repository: https://github.com/opensearch-project/notifications.git
+    working_directory: dashboards-notifications
+    ref: main
```

Comment:
Since notificationsDashboards did not ship with 1.0, take it out. Once #729 is merged and cron runs there will be a 1.2 manifest where you can add it.